### PR TITLE
Documentation for release-2018-08

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: reference-docs/server-configuration/config.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ server: {
 ```js
 editor: {
   // configure the Editor Host. This is used for password reset & login urls.
-  public_host: 'http://localhost:9000'
+  public_host: 'http://localhost:9000',
+  auto_save_interval: 10 // This defines how often the editor auto saves in seconds
```
Comment (DaRaFF) [inline reference-docs/server-configuration/config.md]: @ReicoC `release-2018-08` is an integration branch, and a change also needs a PR. But for this time it's ok :-)

Comment (ReicoC) [reply reference-docs/server-configuration/config.md]: oh sorry, will do so next time

Comment (marcbachmann) [reply reference-docs/server-configuration/config.md]: @DaRaFF Github worked a bit on the branch checks. Maybe they support a regex now which we could use to protect those branches.

Comment (marcbachmann) [reply reference-docs/server-configuration/config.md]: Only one pattern is supported, but we can match against `release*` with a second check
image

**Pull request closed without merging**